crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -3077,15 +3077,56 @@
       },
       "EventType": {
         "description": "Auto-generated discriminant enum variants",
-        "type": "string",
-        "enum": [
-          "MoveEvent",
-          "Publish",
-          "TransferObject",
-          "DeleteObject",
-          "NewObject",
-          "EpochChange",
-          "Checkpoint"
```

Comment:
I don't know why it was necessary to update the schema here, but it could be because of some updated dependencies.